source/contracts/reporting/Market.sol

```diff
@@ -166,6 +192,8 @@ contract Market is DelegationTarget, Typed, Initializable, Ownable, IMarket {
     function updateTentativeWinningPayoutDistributionHash(bytes32 _payoutDistributionHash) public returns (bool) {
         var (_firstPlaceHash, _secondPlaceHash) = MarketExtensions(controller.lookup("MarketExtensions")).getOrderedWinningPayoutDistributionHashes(this, _payoutDistributionHash);
 
+        require(_firstPlaceHash != bytes32(0));
+        require(_firstPlaceHash != _secondPlaceHash);
```

Comment:
In what scenarios can this happen?  It feels like if `getOrderedWinningPayoutDistributionHashes` returns first == second then there is a bug (with the exception of when it returns `0,0`, which is acceptable)

Reply:
There was a bug that let this happen so I added this to safegaurd. We should be able to remove it now but if something results in this case it will be a hard non obvious big to solve